src/query.ts

```diff
@@ -14,33 +14,39 @@ const distribution = 'distribution';
 const dateCreated = 'dateCreated';
 const datePublished = 'datePublished';
 const dateModified = 'dateModified';
-const language = 'language';
-const source = 'source';
-const keyword = 'keyword';
+const language = 'inLanguage';
+const source = 'isBasedOn';
+const keyword = 'keywords';
```

Comment:
These changes are unnecessary, as they are just the variable names, not the actual predicates used. So please revert them.